conftest.py

```diff
@@ -3,6 +3,14 @@
 import shutil
 import stat
 import tempfile
+from sys import platform
+
+# Windows and OSX use spawn instead of fork by default
+# which causes the config server to fail to start
+if platform == "darwin" or platform == "win32":
```

Comment:
I think you could make the argument that this belongs in the config server code in panoptes-utils too. @wtgee let me know if you'd rather see it over there.